scripts/sync-release-branch.sh

```diff
@@ -48,6 +48,7 @@ if [[ -z "$GH_TOKEN" ]]; then
   if [[ -x "$roxhelp_bin" ]]; then
     workflow_root="$(dirname "$roxhelp_bin")/.."
     if [[ -f "${workflow_root}/lib/github.sh" ]]; then
+      # **TODO before merge** What and where is this?
       source "${workflow_root}/lib/github.sh"
```

Comment:
@gavin-stackrox do you know what this is? And how we might be able to get shellcheck to source it?

Reply:
> @gavin-stackrox do you know what this is? And how we might be able to get shellcheck to source it?

Thats the stackrox/workflow repo. I don't know if this script is still live. Maybecheck with release channel.